utils/base_test.py

```diff
@@ -0,0 +1,29 @@
+import os
+import unittest
+from selenium import webdriver
+from webdriver_manager.chrome import ChromeDriverManager
+from selenium.webdriver.chrome.service import Service
+
+
+class BaseTest(unittest.TestCase):
+    def _setUp(self, url=None) -> None:
+        service = Service(ChromeDriverManager().install())
+        chrome_options = webdriver.ChromeOptions()
+        chrome_options.add_argument('--window-size=1920,1080')
+        if 'CICD_RUN' in os.environ:
+            chrome_options.add_argument('--no-sandbox')
+            chrome_options.add_argument('--disable-gpu')
+            chrome_options.add_argument('--disable-dev-shm-usage')
+            chrome_options.add_argument('--headless')
+        self.driver = webdriver.Chrome(service=service, options=chrome_options)
+        if url:
+            self.driver.get(url)
+        else:
+            raise RuntimeError('URL is not defined')
```

Comment:
if URL is a mandatory precondition for the test to work, perhaps just make it a non-optional argument instead of using if/else here?